src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -600,6 +593,11 @@ export default {
 							<slot class="app-sidebar-header__background" name="header" />
 						</div>
 
+						<!-- a11y fallback for empty content -->
+						<NcAppSidebarHeader v-if="empty"
+							class="hidden-visually"
+							:name
+							:tabindex="-1" />
```

Comment:
It should be outside the info block, as the entire info is supposed to be hidden in an empty state